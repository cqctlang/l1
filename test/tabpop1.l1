
tab = { "a" : 1, "b" : 2, "c" : 3 };
while(length(tab)){
	@local v;
	v = pop(tab);
	printf("%a -> %a\n", v[0], v[1]);
}


