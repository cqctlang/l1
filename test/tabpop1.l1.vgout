entry: var params, 9 locals, 0 temps
a -> <cval 1>
c -> <cval 3>
b -> <cval 2>
