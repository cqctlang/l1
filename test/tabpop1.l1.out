entry: var params, 8 locals, 0 temps
"a" -> <cval 1>
"c" -> <cval 3>
"b" -> <cval 2>
